### PR TITLE
Making the site usable for all screen sizes

### PR DESCRIPTION
**Pull request by Mzaien**
The site had some weird css bugs that made it unusable form mobile devices 
**before:** 
CleanShot 2021-04-04 at 09 09 00@2x
**after:**
CleanShot 2021-04-04 at 08 57 34@2x
CleanShot 2021-04-04 at 08 57 45@2x
CleanShot 2021-04-04 at 08 57 52@2x

### CODE REVIEW

**Comment by Mzaien** [comment]
Another small edit to fix Github badge 
CleanShot 2021-04-04 at 15 38 58@2x

**Review by styfle — COMMENTED** [review]
Amazing work thanks! 

Any reason why the indentation was changed on all the lines?

File: public/style.css [hunk 1]
```diff
@@ -1,317 +1,357 @@
...
 }
 
 a {
-    cursor: pointer;
-    color: #0076FF;
-    text-decoration: none;
-    transition: all 0.2s ease;
-    border-bottom: 1px solid white;
+  cursor: pointer;
+  color: #0076ff;
```
Comment (styfle) [inline public/style.css:12]: Why is the color changed here?

**Comment by Mzaien** [comment]
> Amazing work thanks!
> 
> Any reason why the indentation was changed on all the lines?

i'm using Vscode prettier and it made this change and also done this to colors
CleanShot 2021-04-04 at 23 20 48@2x

they are the same colors but characters are small,
so sorry for that i did not mean to change the format of the code or something 😅

**Comment by styfle** [comment]
Let's revert the unintentional changes so we can merge this 👍

**Comment by Mzaien** [comment]
Sorry it took me some time to bring back everything together, but now it is more responsive and even for iPhone 5/se 
CleanShot 2021-04-05 at 00 18 00@2x

File: public/style.css [hunk 2]
```diff
@@ -288,6 +300,12 @@ img {
     .field {
         margin: 20px 10px;
     }
+    .pull-left {
+          min-width: 50%;
+          display: flex;
+          justify-content: flex-end;
+          margin-left: 1.5em;
```
Comment (styfle) [inline public/style.css:307]: I think this is mostly redundant. All you need here is `margin-left` here.

Comment (styfle) [inline public/style.css:307]:
```suggestion
    .pull-left {
          margin-left: 1.5em;
```

I think this is mostly redundant. All you need here is `margin-left` here.

Comment (Mzaien) [reply public/style.css]: Done

**Review by styfle — APPROVED** [review]
Great work, thanks! 🥇

**Pull request merged**